### PR TITLE
ISSUE-152: Find a better way to update Drupal config/sync/core.extension.yml when upgrading

### LINKED ISSUE #152: Find a better way to update Drupal config/sync/core.extension.yml when upgrading
# What?

[Step 5](https://github.com/esmero/archipelago-deployment/blob/1.0.0-RC3/docs/upgradeFromD8ToD9.md#step-5-1) of the upgrade section in the D8 to D9 docs is not an ideal way to avoid the profile config issue when importing. It would be better to find a less error-prone way than asking users to manually edit the file.

### PR DESCRIPTION
**Pull request by aksm**
Resolves #152 and also has some minor spelling/grammar/formatting edits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,28 +2,28 @@
...
 
-This repository serves as bootstrap for a Archipelago 1.0.0-RC3 deployment via Docker using and provides this time many more options:
+This repository serves as bootstrap for a Archipelago 1.0.0-RC3 deployment via Docker and provides many more options this time:
 
 - minio.io (latest) S3/Azure/Local/Remote alternative with a new Console and also Gateway to many others.
 - Apache Solr 8.8.2 (or 7.5 if you go legacy) with the wizardly Solr OCR Highlight library [v0.7.1](https://github.com/dbmdz/solr-ocrhighlighting/releases/tag/0.7.1) build by the Developement Team at the [Bavarian State Library](https://github.com/dbmdz). Thanks Johannes Baiter!
 - MySQL 8.0.22 (or MySQL 5.7 if you go legacy)
 - NGINX 11
-- Custom PHP FPM 7.4 zuper-tunned for Drupal 8/9 with multi-arch (so M1 too) capabilities (or 7.3 FPM if you go legacy) with WARC to WACZ processing and Composer 2.0
+- Custom PHP FPM 7.4 super-tuned for Drupal 8/9 with multi-arch (so M1 too) capabilities (or 7.3 FPM if you go legacy), WARC to WACZ processing, and Composer 2.0
```
Comment (DiegoPino) [inline README.md:11]: The z is on purpose here :)

File: README.md [hunk 2]
```diff
@@ -2,28 +2,28 @@
...
 - Cantaloupe 4.1.9 as IIIF Server with Video Frame extraction and PDF support
 - A Skeleton Project setup to run latest Version of Drupal (9.2.9) and Strawberry Field modules on 1.0.0-RC3 & friends on 0.2.0
 - Official *Drupal 9 ready* release again. D9.2.9 this time. Drupal 8 is no longer maintained, and we have [documented the upgrade path](docs/upgradeFromD8ToD9.md)
 - Complete support for Apple Silicon *M1* Machines and in general arm64 architecture Chips like Raspberry Pi 4, with specially built arm64 docker containers. Blazing fast OCR!
 
-The skeleton project contains all the pieces needed to run a local deployment of a vanilla Archipelago including (*YES*!) content provided as optional feature from [archipelago-recyclables](https://github.com/esmero/archipelago-recyclables)
+The skeleton project contains all the pieces needed to run a local deployment of a vanilla Archipelago including (*YES*!) content provided as an optional feature from [archipelago-recyclables](https://github.com/esmero/archipelago-recyclables)
 
-# Starting from CERO
+# Starting from ZERO
```
Comment (DiegoPino) [inline README.md:20]: Same here. Cero as in spanish, give me some credit! Hahahha

File: README.md [hunk 3]
```diff
@@ -2,28 +2,28 @@
...
+This is the recommended, simplest way for this release. There are a too many, tons of fun new features, Metadata Displays, viewers, new JS libraries, better HUGE import capabilities, bug fixes (yes), webform elements and improvements! W3C WebAnnotations capabilities are even better now by the great team behind <https://recogito.github.io/annotorious/api-docs/osd-plugin/> (thanks Rainer Simon) and with complete integration in our UI and metadata backends. Also [WACZ! and replay.web](https://github.com/webrecorder/replayweb.page) both fabulous developments by the Webrecorder team (thanks Ilya Kreymer and Emma Dickson). There is more. Release will have a full feature list. Promise.
 
-## OSX Intel or Apple Silicon M1:
+## macOS Intel or Apple Silicon M1:
 
-[Step by Step deployment on OSX](docs/osx.md)
+[Step by Step deployment on macOS](docs/osx.md)
```
Comment (DiegoPino) [inline README.md:26]: Lets keep osx? Macos is not really osx

File: docs/upgradeFromD8ToD9.md [hunk 4]
```diff
@@ -36,22 +35,22 @@ docker ps
...
 
-Now let's tar.gz the whole ensemble with data and configs. As an example we will save this into your `$HOME` folder. 
-As a good practice we append the **current date **(YEAR-DAY-MONTH) to the filename. Here we assume today is December 1st of 2021.
+Now let's `tar.gz` the whole ensemble with data and configs. As an example we will save this into your `$HOME` folder. 
+As a good practice we append the **current date **(YEAR-MONTH-DAY) to the filename. Here we assume today is December 1st of 2021.
 
 ```Shell
 cd ..
-sudo tar -czvpf $HOME/archipelago-deployment-RC2-20210112.tar.gz archipelago-deployment
+sudo tar -czvpf $HOME/archipelago-deployment-RC2-20211201.tar.gz archipelago-deployment
```
Comment (DiegoPino) [inline docs/upgradeFromD8ToD9.md:43]: I do not want to have the date formar conversation but ok, we can roll with this change

File: docs/upgradeFromD8ToD9.md [hunk 5]
```diff
@@ -98,7 +98,7 @@ sudo chmod 666 web/sites/default/*services.yml
 
 ### Step 2:
 
-Time to fetch `1.0.0-RC3` branch and update our `docker-compose` and `composer` dependencies. We are also going to stop current `Docker` ensemble to update all containers to newer versios. E.g, on OSX run:
+Time to fetch the `1.0.0-RC3` branch and update our `docker-compose` and `composer` dependencies. We are also going to stop current `Docker` ensemble to update all containers to newer versios, e.g. on macOS run:
```
Comment (DiegoPino) [inline docs/upgradeFromD8ToD9.md:101]: No macos please. Osx

**Review by DiegoPino — COMMENTED** [review]
Hi, thanks for this. There are really many many changes here so will better wait for @alliomeria to come back. I agree with 90% of those, some errors are actually on purpose and osx thing that is not macos but might be persuaded when back in front of my computer. Monday merge if @alliomeria agrees? If not wednesday Thanks!

**Comment by aksm** [comment]
@DiegoPino Glad to reconsider/revert as my edits might be the result of my misunderstandings. In any case, sounds good to wait to review together.

**Comment by DiegoPino** [comment]
I like them all no worries (I know my use of Zuper and Cero are confusing, I'm not really a very serious human). But I won't have a good insight today. No need to revert, let's let all sit a bit and let @alliomeria jump in too. Thanks so much, really, this is really good

**Pull request merged**